io.github.bkbilly.lnxlink.yaml

```diff
@@ -0,0 +1,165 @@
+id: io.github.bkbilly.lnxlink
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: lnxlink
+finish-args:
+  # Wayland access
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  # For DBus
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --system-talk-name=org.freedesktop.login1
+  - --system-talk-name=org.freedesktop.UPower
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gnome.Mutter.IdleMonitor
+  - --talk-name=org.mpris.MediaPlayer2.*
```

Comment:
Why is this needed? Is this a MPRIS client?

Reply:
This is used to get information from the current playing media players: https://github.com/bkbilly/lnxlink/blob/master/lnxlink/modules/media.py